MAINTAINERS.md

```diff
@@ -0,0 +1,14 @@
+# Kubernetes Event-driven Autoscaling (KEDA) Maintainers
+
+## Maintainers
+
+| Maintainer           | GitHub ID                                     | Affiliation |
+| -------------------- | --------------------------------------------- | ----------- |
+| Jeff Hollan          | [jeffhollan](https://github.com/jeffhollan)   | Microsoft   |
+| Anirudh Garg         | [anirudhgarg](https://github.com/anirudhgarg) | Microsoft   |
+| Aarthi Saravanakumar | [Aarthisk](https://github.com/Aarthisk)       | Microsoft   |
+| Yaron Schneider      | [yaron2](https://github.com/yaron2)           | Microsoft   |
+| Ahmed ElSayed        | [ahmelsayed](https://github.com/ahmelsayed)   | Microsoft   |
+| Zbynek Roubalik      | [zroubalik](https://github.com/zroubalik)     | Red Hat     |
+| Ben Browning         | [bbrowning](https://github.com/bbrowning)     | Red Hat     |
```

Comment:
```suggestion
```

Reply:
Is that ok for you @zroubalik?

Reply:
Introduced alumni and added @bbrowning to that but will wait for @bbrowning or @zroubalik to agree on that.